### PR TITLE
refactor(editor): Migrate `AuthView` and associated components to composition api

### PR DESCRIPTION
**Pull request by MiloradFilipovic**
## Description
Moving `AuthView` and associated components to composition api:
- `AuthView.vue`
- `ChangePasswordView.vue`
- `ForgotMyPasswordView.vue`
- `SamlOnboarding.vue`
- `SetupView.vue`
- `SigninView.vue`

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/views/ChangePasswordView.vue [hunk 1]
```diff
@@ -1,164 +1,166 @@
...
-		if (mfaEnabled) {
-			form.inputs.push({
-				name: 'mfaToken',
-				initialValue: '',
+const password = ref('');
+const loading = ref(false);
+const config = ref<IFormBoxConfig | null>(null);
+
+onMounted(async () => {
+	console.log('onMounted');
```
[the remark this answers is not in the document]
Comment (RicardoE105) [reply packages/editor-ui/src/views/ChangePasswordView.vue:25]: Nitpick: We should follow the order:

1. Props and Emits
2. Stores
3. Composables
4. Data
5. Computed
6. Methods
7. Watchers
8. Lifecycle

File: packages/editor-ui/src/views/SetupView.vue [hunk 2]
```diff
@@ -1,129 +1,123 @@
...
-				} else {
-					await this.$router.push({ name: VIEWS.USERS_SETTINGS });
-				}
-			} catch (error) {
-				this.showError(error, this.$locale.baseText('auth.setup.settingUpOwnerError'));
-			}
-			this.loading = false;
+const posthogStore = usePostHog();
+const settingsStore = useSettingsStore();
+const uiStore = useUIStore();
```
Comment (RicardoE105) [inline packages/editor-ui/src/views/SetupView.vue:24]: nitpick: follow order :
1. Props and Emits
2. Stores
3. Composables
4. Data
5. Computed
6. Methods
7. Watchers
8. Lifecycle

File: packages/editor-ui/src/views/SigninView.vue [hunk 3]
```diff
@@ -1,191 +1,207 @@
...
+import { MFA_AUTHENTICATION_REQUIRED_ERROR_CODE, VIEWS, MFA_FORM } from '@/constants';
+
+const route = useRoute();
+const router = useRouter();
+
+const toast = useToast();
+const locale = useI18n();
+const telemetry = useTelemetry();
+
+const usersStore = useUsersStore();
```
Comment (RicardoE105) [inline packages/editor-ui/src/views/SigninView.vue:26]: nitpick: following order mentioned in the other comments

**Review by RicardoE105 — COMMENTED** [review]
Nice work 🙏 thanks very much

**Review by RicardoE105 — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.61.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.61.0)

**Pull request merged**